components/selection/selection-summary.js

```diff
@@ -50,10 +50,13 @@ class Summary extends LocalizeCoreElement(SelectionObserverMixin(LitElement)) {
 	}
 
 	_updateSelectSummary() {
-		if (this._provider && this._provider.selectionSingle) return;
+		if (this._provider?.selectionSingle) {
+			this._summary = null;
+			return;
+		}
```

Comment:
This is the main change here - making sure `_summary` is always `null` for single selection.

Reply:
I think there is a slight behaviour change here. Previously we would have proceeded when `_provider` was `null` or `undefined`, whereas now we would not. I _think_ that's ok, and should allow us to simplify the conditional statements below.